RecoTauTag/Configuration/test/ZTT-validation.txt

```diff
@@ -1,2 +1 @@
-/store/relval/CMSSW_9_0_0_pre4/RelValTTbar_13/GEN-SIM-RECO/PUpmx25ns_90X_mcRun2_asymptotic_v1-v1/10000/5C714DBC-65EE-E611-BCFB-0CC47A78A426.root
-/store/relval/CMSSW_9_0_0_pre4/RelValTTbar_13/GEN-SIM-RECO/PUpmx25ns_90X_mcRun2_asymptotic_v1-v1/10000/A40BB4B4-65EE-E611-A3C2-0CC47A7C3412.root
+/store/relval/CMSSW_8_0_0/RelValTTbar_13/GEN-SIM-RECO/PU25ns_80X_mcRun2_asymptotic_v4-v1/10000/42D6DF66-9DDA-E511-9200-0CC47A4D7670.root
```

Comment:
Is it still meaningful to set input EDM files from CMSSW_8_0_0 (it looks like going backwards)?
IIUC, this file (RecoTauTag/Configuration/test/ZTT-validation.txt) is used in some private validation testing.
So, I just need a confirmation that this is not made by some mistake.